cmd/mrinfo.cpp

```diff
@@ -274,6 +276,7 @@ void run ()
     else if (export_grad || check_option_group (GradImportOptions) || dwgrad || shell_bvalues || shell_sizes)
       DWI::set_DW_scheme (header, DWI::get_valid_DW_scheme (header, true));
 
+    if (name)       std::cout << argument[i] << "\n";
```

Comment:
Note that in some instances this will give a different result to `header.name()`. Would the latter make more sense, i.e. as with other options, providing the header data "as *MRtrix3* sees it"?

Reply:
Not sure what the instances are in which there is a difference between argument and name but `header.name()` is now probably more in line with the option name.

Reply:
The main reason `header.name()` might differ from the argument given is with DICOM data: in this case, the `header.name()` is formed from the DICOM headers. 

Personally, I'm happy either way, though.

Reply:
I think was actually referring to number sequence usage; looks like it comes out the same though.